### ISSUE TITLE
Stable documentation is not deployed

### ISSUE THREAD

**Issue by hyrodium** [ohno/Antique.jl#4]
https://ohno.github.io/Antique.jl/stable returns 404.
Maybe, `DOCUMENTER_KEY` is not set correctly?
https://documenter.juliadocs.org/dev/man/hosting/#GitHub-Actions

![image](https://github.com/ohno/Antique.jl/assets/7488140/9bad6500-bea6-4213-bc8a-314fb22272ea)

[JuliaRegistrator や TagBot でtag 打ちをしても stable Document が生成されない対処法](https://zenn.dev/terasakisatoshi/articles/87e730a50915f9)

**Comment by ohno** [comment]
Currently https://ohno.github.io/Antique.jl/stable/ is working. I have not changed `DOCUMENTER_KEY`. This might be due to that I deleted the `gh-pages` branch after adding the tag (v0.1.0). I deleted it because it did not work after the package name change. Just now, the stable documentation is automatically generated (https://github.com/ohno/Antique.jl/commit/e505d20feb392c9794bc4cce97d764a10d0e9dbc) by what I added a new tag (v0.1.1).
